docs/scripts_js/EXAMPLE_CreateAssayFile.js

```diff
@@ -16,6 +16,10 @@ import * as arctrl from "@nfdi4plants/arctrl"
 
 // Import ARCtrl Assay to Spreadsheet transformation
 import {toFsWorkbook,fromFsWorkbook} from "@nfdi4plants/arctrl/ISA/ISA.Spreadsheet/ArcAssay.js"
+// Import ARCtrl Study to Spreadsheet transformation
```

Comment:
Maybe we can move this different modes of import into an `import.js`?

Reply:
ah wrong place. I meant namespace or direct import

Reply:
Docs I meant

Reply:
so you mean, move them into an import.js so we can import them from the import.js? 😅  i would rather tackle this issue at its roots.